src/pykeen/evaluation/classification_evaluator.py

```diff
@@ -116,3 +151,26 @@ def finalize(self) -> ClassificationMetricResults:  # noqa: D102
         self.all_scores.clear()
 
         return ClassificationMetricResults.from_scores(y_true, y_score)
+
+
+def construct_indicator(*, y_score: np.ndarray, y_true: np.ndarray) -> np.ndarray:
+    """Construct binary indicators from a list of scores.
```

Comment:
What is this method doing?

Reply:
It's the one that takes the list of scores for each triple and binarizes it into 0/1. Tbh I don't know how it works or what this process is called, I just adapted it from the other package (NetMF) that @lvermue linked

Reply:
Did you test this part? E.g., the outer loop seems to always make one iteration, and some of the reshaping looks superfluous to me.

```python
y_score_m = y_score.reshape(1, -1)
y_true_m = y_true.reshape(1, -1)
num_label = np.sum(y_true_m, axis=1, dtype=int)
y_sort = np.fliplr(np.argsort(y_score_m, axis=1))
y_pred = np.zeros_like(y_true_m, dtype=int)
for i in range(y_true_m.shape[0]):  # -> y_true_m.shape[0] = 1
    for j in range(num_label[i]):
        y_pred[i, y_sort[i, j]] = 1
return y_pred.reshape(-1)
```